modules/nw-networkpolicy-deny-all-allowed.adoc

```diff
@@ -60,23 +58,23 @@ kind: NetworkPolicy
 apiVersion: networking.k8s.io/v1
 metadata:
   name: deny-by-default
-  namespace: default <1>
+  namespace: my-project <1>
 spec:
   podSelector: {} <2>
   ingress: [] <3>
 endif::multi[]
 ----
 ifdef::multi[]
-<1> `namespace: default` deploys this policy to the `default` namespace.
+<1> `namespace: my-project` deploys this policy to the `my-project` namespace.
 <2> `network_name`: specifies the name of a network attachment definition.
-<3> `podSelector:` is empty, this means it matches all the pods. Therefore, the policy applies to all pods in the default namespace.
+<3> `podSelector:` is empty, this means it matches all the pods. Therefore, the policy applies to all pods in the `my-project` namespace.
 <4> `policyTypes:` a list of rule types that the `NetworkPolicy` relates to.
 <5> Specifies as `Ingress` only `policyType`.
 <6> There are no `ingress` rules specified. This causes incoming traffic to be dropped to all pods.
 endif::multi[]
 ifndef::multi[]
-<1> `namespace: default` deploys this policy to the `default` namespace.
-<2> `podSelector:` is empty, this means it matches all the pods. Therefore, the policy applies to all pods in the default namespace.
+<1> `namespace: my-project` deploys this policy to the `my-project` namespace.
+<2> `podSelector:` is empty, this means it matches all the pods. Consequently, the policy applies to all pods in the `my-project` namespace.
```

Comment:
Ideally, avoid starting sentences with stuff in backticks (I know it was already that way). Can these be updated to something like:

<1> Specify the namespace in which to deploy this policy.
<2> If this field is empty, the policy matches all pods and applies to all pods in the specified namespace.
<3> If this field is empty, incoming traffic to all pods is dropped.